config-model/src/main/java/com/yahoo/vespa/model/container/search/RpcResourcePoolComponent.java

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.model.container.search;
+
+import com.yahoo.osgi.provider.model.ComponentModel;
+import com.yahoo.vespa.model.container.component.Component;
+import com.yahoo.vespa.model.container.xml.BundleMapper;
+
+public class RpcResourcePoolComponent extends Component<RpcResourcePoolComponent, ComponentModel> {
+
+    public RpcResourcePoolComponent() {
+        super(toComponentModel());
+    }
+
+    private static ComponentModel toComponentModel() {
+        String className = "com.yahoo.search.dispatch.rpc.RpcResourcePool";
```

Comment:
container-search is available from the model, so the class object can be used instead of a string. It will then fail compile-time instead of runtime if the class is moved or renamed.

Reply:
Thanks, done.